### PR TITLE
Fixes issue108

### PR DESCRIPTION
**Pull request by zeusdeux**

### CODE REVIEW

**Comment by zeusdeux** [comment]
Gotta amend my commit messages. Forgot that I was opening a PR for a different repo. Damn.

**Comment by zeusdeux** [comment]
Squashed and fixed commit message.

@zeke does this work?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,44 @@
...
+- package name *can* consist of hyphens
+- package name must *not* contain any non-url-safe characters (since name ends up being part of a URL)
+- package name should not start with `.` or `_`
+- package name should *not* contain any leading or trailing spaces
+- package name *cannot* be the same as a node.js/io.js core module nor a reserved/blacklisted name. For example, the following names are invalid:
+    + http
+    + stream
+    + node_modules
+    + favicon.ico
+- package name length cannot exceed 50 i.e., package name cannot contain more than 50 characters
```
Comment (iarna) [inline README.md]: change 50 to 214

**Comment by zkat** [comment]
Fixed as of 3af92c8

**Pull request closed without merging**